kod/object/active/holder/room.kod

```diff
@@ -4888,6 +4980,120 @@ messages:
    {
       return $;
    }
+   
+   GetPlayers()
+   {
+      local i, plPlayers;
```

Comment:
Should only use the "p" nomenclature for properties. These message are a great idea, I'm wondering if we would use them frequently enough to warrant making them property lists and adding/removing whenever players/monsters enter/leave... can worry about that later though (i.e. if we start using them often in other non-event rooms).